### PR TITLE
Fixes #649: Throws audio level message when user does not speak.

### LINKED ISSUE #649: Implement check audio levels message when user doesn't speak anything for a while.
Google gives this check audio levels message when it is not able to receive any audio signal.
Implement this feature to susper
**Screenshots of the issue**

image

### PR DESCRIPTION
**Pull request by harshit98**
Fixes issue #649 

Changes: 
- now when a user does not speak anything, it throws an audio level message.

Demo Link: https://harshit98.github.io/susper.com/

Kindly review @nikhilrayaprolu @Marauderer97

### CODE REVIEW

File: src/app/speechtotext/speechtotext.component.ts [hunk 1]
```diff
@@ -86,7 +86,8 @@ export class SpeechtotextComponent implements OnInit {
 
       if (t === 5) {
         this.subscription.unsubscribe();
-        this.store.dispatch(new speechactions.SearchAction(false));
+        this.message = "Please check your microphone and audio levels.";
+        // this.store.dispatch(new speechactions.SearchAction(false));
```
Comment (nikhilrayaprolu) [inline src/app/speechtotext/speechtotext.component.ts]: great work @harshit98 uncomment this line. but dispatch the action when t==7.
this action automatically closes the speech UI and returns to the index page.

**Comment by harshit98** [comment]
@nikhilrayaprolu I didn't get it. Why do we need to change microphone color? Already it's white following market leader. :)

**Comment by harshit98** [comment]
Okay I got it @nikhilrayaprolu , you mean when it throws message "Please check audio level.." the color of microphone should change to grey. Working on it.

File: src/app/speechtotext/speechtotext.component.ts (hunk 1 above)
Comment (harshit98) [inline src/app/speechtotext/speechtotext.component.ts]: @nikhilrayaprolu but using `if (t==6)` condition after `if (t==4)` is not being executed. I tried dispatching action at `t==6` but it does not works.

**Comment by harshit98** [comment]
Done @nikhilrayaprolu

**Pull request merged**